index.html

```diff
@@ -7,80 +7,86 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <!-- 네비게이션 -->
-    <header class="nav-bar">
-      <a href="'/'" target="_blank">
-        <img  class="logo" src="Property 1=sm.png">
-      </a>
-      <a href="/login" target="_blank">
-        <img class="login-btn" src="btn_small.png">
-      </a>  
-    </header>
-  <!--본문 시작-->
-  <figure>
-    <div class="home-top">
-      <p class="words"> 일상의 모든 물건을<br>거래해 보세요</p>
-      <img src="Img_home_top.png" class="top-image">
-      <a href="/items" target="_blank">
-        <img class="show-more-btn" src="btn_large.png" alt="구경하러 가기">
-      </a>
-    </div>
-  </figure>
-  <figure class="first-p">
-      <img class="hot-item" src="desktop_02.png">
-  </figure>
-  <figure class="second-p">
-      <img class="search-item" src="desktop_03.png">
-  </figure>
-  <figure class="third-p">
-      <img class="resister-item" src="desktop_04.png">
-  </figure>
-<section>
-  <div class="home-bottom">
-    <p class="bottom-word">믿을 수 있는<br> 판다마켓 중고 거래</p>
-    <img src="Img_home_bottom.png" class="bottom-img">
-  </div>  
-</section>
-
-<!--푸터 시작-->
-<footer class="footer-style">
-  <span class="footer-inner">
-    <span class="codeit-2024">@codeit - 2024</span>
-
-    <!--Privacy Policy , FAQ 시작-->
-    <span class="footer-link">
-      <a href="/privacy">
-        <span class="Policy">Privacy Policy</span>
-      </a>
-      <a href="/faq">
-        <span class="FAQ">FAQ</span>
+  <!-- header 시작 -->
+  <header class ="header">
+    <a href= "'/'">
+      <img src="../images/Property 1=sm.png" alt="판다마켓" class="header_img"></a>
+    <a href="login.html" target="_blank" class="login_btn">
+      <img src="../images/btn_small.png" alt="btn_small.png" class="login">
+    </a>
+  </header>
+  <main>
+    <!-- 일상의 모든 물건-->
+    <section class="top_banner">
+      <div class="banner_1">
+        <div class="banner-text">
+          일상의 모든 물건을<br>
+          거래해보세요
+        </div>
+      <a href="/items" target="blank" class="more_btn">
+        <img src="../images/btn_large.png" alt="btn_large.png" class="morebtn">
       </a>
-    </span>
+        <img src="../images/Img_home_top.png" class="topbannerimg">
+      </div>
+    </section>
+    <section class="paragraph">
+      <!-- 인기상품 -->
+      <div class ="para1">
+        <img src="../images/Img_home_01.png" alt="para1-img" class="pimg1">
+         <div class="para-text">
+          <h5 class="smtx">Hot item</h5><br>
```

Comment:
❗️ 수정요청
h5태그와 h3태그는 block 요소라서 태그 사이에 줄바꿈 태그인 br 태그가 불필요합니다~
불필요한 태그는 유지보수 및 가독성에 좋지 않으니 삭제 추천드려요!

```suggestion
          <h5 class="smtx">Hot item</h5>
```